tests/integrations/test_dbm.py

```diff
@@ -135,6 +135,8 @@ def setup_dbm_comment(self):
         self.r = weblog.get("/stub_dbm", params={"integration": self.integration, "operation": self.operation})
 
     def test_dbm_comment(self):
+        print(self.r.status_code)
```

Comment:
in `utils.tools`, you have `logger` than allow you to debug print.

Reply:
noted